sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_entities_async.py

```diff
@@ -121,6 +121,14 @@ async def test_input_with_all_errors(self, client):
         self.assertTrue(response[1].is_error)
         self.assertTrue(response[2].is_error)
 
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer()
+    async def test_too_many_documents(self, client):
+        docs = ["One", "Two", "Three", "Four", "Five", "Six"]
+
+        with self.assertRaises(HttpResponseError):
```

Comment:
should we add a check to ensure that the status code is 400?

Reply:
don't see why not, adding the test, thanks!